corehq/apps/domain/templates/login_and_password/login.html

```diff
@@ -1,10 +1,26 @@
 {% extends "style/base_page.html" %}
 {% load i18n %}
+{% load hq_shared_tags %}
 
 {% block title %}{% trans "Log into CommCare HQ" %}{% endblock title %}
 
 {% block page_content %}
     {% block login-content %}
     {% include "login_and_password/partials/login_full.html" %}
     {% endblock %}
+
+    {% if implement_password_hashing %}
+        <script src="{% static 'domain/js/encoder.js' %}"></script>
+        <script>
+            $(document).ready(function(){
+                (function(){
+                  p_field=$("#id_auth-password");
+                  p_field.parents("form")[0].onsubmit = function(){
+                    p_encoder = new HexParsr("{{psalt1}}","{{psalt2}}");
+                    p_field.val(p_encoder.xyz(p_field.val()))
+                  };
+                })()
+            });
+        </script>
```

Comment:
Moving this to `encoder.js` itself along with logic to generate these values.
I added it like this thinking that static js should not contain any dynamic content since they are precompiled and stored. Guess thats not true.